tests/Aspire.Playground.Tests/AppHostTests.cs

```diff
@@ -0,0 +1,218 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net;
+using System.Text.Json;
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Workload.Tests;
+using Microsoft.Extensions.DependencyInjection;
+using SamplesIntegrationTests;
+using SamplesIntegrationTests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Aspire.Playground.Tests;
+
+public class AppHostTests
+{
+    private readonly TestOutputWrapper _testOutput;
+    private static readonly string? s_appHostNameFilter = Environment.GetEnvironmentVariable("TEST_PLAYGROUND_APPHOST_FILTER");
+
+    public AppHostTests(ITestOutputHelper testOutput)
+    {
+        this._testOutput = new TestOutputWrapper(testOutput);
```

Comment:
Why do we need this `TestOutputWrapper`? Why can't we use the `ITestOutputHelper` directly?

Reply:
This is useful to see the output streaming to `testOutputHelper` when the test run might be taking long. But in the case of these tests most of the output is being logged to the console anyway, so we can drop this.